09_json/mohitnag/cmd/puppy-server/main_test.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"testing"
+
+	"github.com/anz-bank/go-course/09_json/mohitnag/pkg/puppy"
+	"github.com/anz-bank/go-course/09_json/mohitnag/pkg/puppy/store"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	defaultPuppy     = "./../../pkg/puppy/store/testdata/default-puppy.json"
+	invalidPuppyJSON = "./../../pkg/puppy/store/testdata/invalid-puppy.json"
+	duplicatePuppies = "./../../pkg/puppy/store/testdata/duplicate-puppies.json"
+)
+
+func TestMain(t *testing.T) {
+	assert := assert.New(t)
+	var buf bytes.Buffer
+	out = &buf
+	args = []string{"--data", "./../../pkg/puppy/store/testdata/many-puppies.json"}
```

Comment:
nit: this path belongs with the other as a `const` further up. Keeping them together would make it easier if in future the source code were to be restructured.

Reply:
Done